core/unit_test/TestReduce.hpp

```diff
@@ -635,7 +635,7 @@ TEST(TEST_CATEGORY, int_combined_reduce_mixed) {
 
   uint64_t nsum = (nw / 2) * (nw + 1);
 
-  auto result1_v = Kokkos::View<int64_t, Kokkos::HostSpace>{"result1_v"};
+  auto result1_v = Kokkos::View<int64_t>{"result1_v"};
```

Comment:
Why did you change the test.  This looks intentional.

Reply:
It is intentional.

Reply:
I changed it to demonstrate the issue that I encountered in #4889 that was failing with an incompatible View assignment error. Is there are reason to only allow `HostSpace` here?

Reply:
Oh I see. Can we change the test to take four reduction args, i.e. result1_v with defaultspace, result2_v with HostSpace, result3 as a scalar, and a fourth using the reducers (which is wrapping a host view)?